iotfunctions/util.py

```diff
@@ -1080,7 +1080,7 @@ def write_usage(self, db, start_ts=None, end_ts=None):
                 start_ts = end_ts - dt.timedelta(seconds=elapsed)
 
             if result > 0:
-                entry = {"entityTypeName": self.parent.name, "kpiFunctionName": i.get('created_by', 'unknown'),
+                entry = {"entityTypeName": self.parent.name, "kpiFunctionName": i.get('kpi_function_name', 'unknown'),
```

Comment:
Same like previous comment. use create_by if there is no kpi_function_name for any stage.